shared/getMetrics.js

```diff
@@ -219,11 +219,11 @@ const getMetrics = ({
         testPlanReport?.runnableTests?.length || countTests({ ...result });
     const testsFailedCount = testsCount - testsPassedCount;
 
-    const requiredFormatted = `${requiredAssertionsPassedCount} of ${requiredAssertionsCount} passed`;
+    const requiredFormatted = `${mustAssertionsPassedCount} of ${mustAssertionsCount} passed`;
     const optionalFormatted =
```

Comment:
I think `requiredFormatted` needs to be `mustFormatted` and `optionalFormatted` needs to be `shouldFormatted`, right?

Reply:
Ok done!